packages/app/cypress/e2e/navigation.cy.ts

```diff
@@ -2,7 +2,7 @@ import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
 
 describe('Navigation', () => {
   before(() => {
-    cy.scaffoldProject('todos')
+    cy.scaffoldProject('todos', { timeout: 50 * 1000 })
```

Comment:
Would really like to understand why this only affects `todos`, if it crops up again let's do more investigation before bumping timeouts.